versioned_docs/version-latest/protocol/substrate-cli.mdx

```diff
@@ -404,7 +379,7 @@ We will be downloading two files for your respective operating system.
 
 </TabItem>
 
-<TabItem value="linux" label="🐧Ubuntu">
+<TabItem value="linux" label="🐧Ubuntu (Home PC)">
```

Comment:
If it were up to me, I'd remove that whole tab altogether. But I thought you probably meant to say that there are users who will put it on their home computer, and that's what the Subspace team is really campaigning for. So that's what I'm leaving it in.

Reply:
my suggestion is this.
Merge ubuntu part and service part into one tab. Then this page will be quite long, but it will be easy to navigate through this page thanks to the tags in the menu (#). 
Here is for example how the menu looks like in my web version.
If someone needs a service, he just clicks on "Systemd servise file" menu item and there it is. 
And so we should make these options as header 3-d level, so that they are submenus of the "Installation" item. By the way, we can even include a template there

Reply:
![Uploading Screenshot_2023-11-12_22-38-45.jpg…]()

Reply:
![Screenshot_2023-11-12_22-38-45](https://github.com/subspace/subspace-docs/assets/77671963/049e9cc0-50d9-450d-9c36-bf702da1ac27)

Reply:
my suggestion is this.
Merge ubuntu part and service part into one tab, equipping simple command with short guide for Tmux/Screen etc. and adding respective 3-d level header "Running node and Farmer in Tmux/Screen" 
 So this page will be quite long, but it will be easy to navigate through this page thanks to the #tags in the menu. 
I created this structure in my local repo, see the screenshot. 

If someone needs a service, he just clicks on "Systemd servise file" menu item and go further
By the way, we can even include a template there
![Screenshot_2023-11-12_23-15-55](https://github.com/subspace/subspace-docs/assets/77671963/961ff92b-c952-4a47-8d97-f922152512af)

Reply:
And is mobile so important when setting up a node?
How about that we can still can make convenient transitions with the help of internal links - # tags ?

Reply:
But you're right in that the instruction where you don't have to skip anything is the most comfortable

Reply:
Your version with different tabs for (select the installation method and then follow the uninterrupted instructions) I'm not sure it's a true universal. It sounds good though.
But I for example choose the installation method after I have read everything. And sometimes I just look for instructions for Docker. So it's not that obvious to everyone.